src/app/command_line_parser.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * @license
+ * Copyright (C) 2020 The Android Open Source Project
```

Comment:
this project is currently licensed under Apache2 - no need to relicense these files.

https://github.com/samccone/twinkie/blob/master/LICENSE

Reply:
Could you please clarify what exactly should be written here?
Should I just replace this line "Copyright (C) 2020 The Android Open Source Project" 
with "Copyright 2020 Google Inc." (like in LICENSE file)?

The instruction (How to apply the Apache License to your work) says:
```
APPENDIX: How to apply the Apache License to your work.
 *
 *       To apply the Apache License to your work, attach the following
 *       boilerplate notice, with the fields enclosed by brackets "[]"
 *       replaced with your own identifying information. (Don't include
 *       the brackets!)  The text should be enclosed in the appropriate
 *       comment syntax for the file format. We also recommend that a
 *       file or class name and description of purpose be included on the
 *       same "printed page" as the copyright notice for easier
 *       identification within third-party archives.
 *
 *    Copyright [yyyy] [name of copyright owner]
 *
 *    Licensed under the Apache License, Version 2.0 (the "License");
 *    you may not use this file except in compliance with the License.
 *    You may obtain a copy of the License at
 *
 *        http://www.apache.org/licenses/LICENSE-2.0
 *
 *    Unless required by applicable law or agreed to in writing, software
 *    distributed under the License is distributed on an "AS IS" BASIS,
 *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 *    See the License for the specific language governing permissions and
 *    limitations under the License.
``` 

As I can see, the only difference with the instruction is a line with copyright.